community/CM-Configuration-Management/policy-openshift-gitops-policygenerator.yaml

```
@@ -0,0 +1,135 @@
+                  kustomizeBuildOptions: --enable-alpha-plugins
+            - complianceType: musthave
+              objectDefinition:
+                kind: ClusterRole
+                apiVersion: rbac.authorization.k8s.io/v1
+                metadata:
+                  name: openshift-gitops-policy-admin
+                rules:
+                  - verbs:
+                      - get
+                      - list
+                      - watch
+                      - create
+                      - update
+                      - patch
+                      - delete
+                    apiGroups:
+                      - policy.open-cluster-management.io
+                    resources:
+                      - policies
+                      - placementbindings
+                  - verbs:
+                      - get
+                      - list
+                      - watch
+                      - create
+                      - update
+                      - patch
+                      - delete
+                    apiGroups:
+                      - apps.open-cluster-management.io
+                    resources:
+                      - placementrules
+                  - verbs:
+                      - get
+                      - list
+                      - watch
+                      - create
+                      - update
+                      - patch
+                      - delete
+                    apiGroups:
+                      - cluster.open-cluster-management.io
+                    resources:
+                      - placements
+                      - placements/status
+                      - placementdecisions
+                      - placementdecisions/status
+            - complianceType: musthave
+              objectDefinition:
+                kind: ClusterRoleBinding
+                apiVersion: rbac.authorization.k8s.io/v1
+                metadata:
+                  name: openshift-gitops-policy-admin
+                subjects:
+                  - kind: ServiceAccount
+                    name: openshift-gitops-argocd-application-controller
+                    namespace: openshift-gitops
+                roleRef:
+                  apiGroup: rbac.authorization.k8s.io
+                  kind: ClusterRole
+                  name: openshift-gitops-policy-admin
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+  name: binding-openshift-gitops-policygenerator
+placementRef:
+  name: placement-openshift-gitops-policygenerator
+  kind: PlacementRule
+  apiGroup: apps.open-cluster-management.io
+subjects:
+  - name: openshift-gitops-policygenerator
+    kind: Policy
+    apiGroup: policy.open-cluster-management.io
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+  name: placement-openshift-gitops-policygenerator
+spec:
+  clusterSelector:
+    matchExpressions:
+      - {key: environment, operator: In, values: ["dev"]}
```

Since this only applies to the hub, can we provide that specific placement instead of our normal default?
```suggestion
      - {key: name, operator: In, values: ["local-cluster"]}
```

---

Good point!